jina/parsers/__init__.py

```diff
@@ -205,13 +205,23 @@ def get_main_parser():
     set_pod_parser(
         sp.add_parser(
             'executor',
-            aliases=['pod'],
+            aliases=['pea'],
```

Comment:
Breaking change? Why pod parser is related to pea cli now? Then what is jina pea cli then?

Reply:
will revert